core/metacling/src/TCling.cxx

```diff
@@ -5891,11 +5869,98 @@ Int_t TCling::AutoParse(const char *cls)
    return nHheadersParsed > 0 ? 1 : 0;
 }
 
+void* TCling::LazyFunctionCreatorAutoloadForModule(const std::string& mangled_name) {
+   using namespace llvm::object;
+   using namespace llvm::sys::path;
+   using namespace llvm::sys::fs;
+
+   static constexpr std::hash<std::string> hash_fn;
+   static std::unordered_map<size_t, uint8_t> libtable;
```

Comment:
Could you be more specific in what these two numbers are? They are not libraries: the name `libtable` is misleading.